README.md

```diff
@@ -4,7 +4,7 @@ Bundle for interfacing UFS models with JEDI interfaces
 
```

Comment:
Please also provide some background information and the link for creating the case-sensitive volume on macOS to compile w/ the patch logic.

Reply:
Done.